sunpy/timeseries/sources/goes.py

```diff
@@ -66,14 +65,16 @@ class XRSTimeSeries(GenericTimeSeries):
     if h5netcdf_version >= packaging.version.parse("0.10"):
         _netcdf_read_kw['decode_vlen_strings'] = True
 
-    def plot(self, axes=None, **kwargs):
+    def plot(self, axes=None, channels=["xrsa", "xrsb"], **kwargs):
```

Comment:
I might have decided this was a good idea.

Reply:
this is a good idea - I guess the only concern is that it isn't consistent across the other `timeseries` sources. Could we make it more generic and have "columns" and then it can be extended to all sources?

Reply:
I am not sure how to make this more general based on the timeseries but it is something we should look at.

Reply:
Yeah, I added this functionality for `GenericTimeseries`, so it would be nice to roll it out to all the sources. I'll open an issue and try and tackle it for 4.0.

Reply:
Ah I forgot about that. I have renamed channels to columns.